packages/appstore/src/components/add-options/add-options.scss

```diff
@@ -5,8 +5,10 @@
     align-items: center;
     max-width: 240px;
     padding: 1rem;
+    height: 60px;
+    width: 240px;
+    border: 2px dashed rgba(255, 68, 79, 0.16);
```

Comment:
ditto
I think we have the variable of the color that you are using. You might find it in our shared/src/styles/constants file

Reply:
We don't have that defined anywhere, we can define it if we need it in more places.

Reply:
I defined` $color-red-6 ` for this, don't change it pls